echopype/echodata/combine.py

```diff
@@ -177,13 +178,19 @@ def combine_echodata(echodatas: List[EchoData], combine_attrs="override") -> Ech
     old_attrs: Dict[str, List[Dict[str, Any]]] = dict()
 
     for group, value in EchoData.group_map.items():
```

Comment:
Just for the sake of readability, seems a good idea to add a line of comment above this block to point to the file (`echodata/convention/1.0.yml`) where people will find the `group_path` names (e.g., `top`, `sonar`, `beam`, etc) corresponding with the actual group names ("Top-level", "Sonar", "Beam_group1", etc)

Reply:
Done. See latest changes.